modules/nf-core/tcoffee/align/tests/main.nf.test

```diff
@@ -168,10 +177,43 @@ nextflow_process {
         then {
             assertAll(
                 { assert process.success },
-                { assert snapshot(process.out.alignment).match("alignment - lib") },
                 { assert path(process.out.lib.get(0).get(1)).getText().contains("1ahl") },
-                { assert snapshot(process.out.versions).match("versions_lib") }
+                { assert snapshot(
+                                    process.out.alignment,
+                                    process.out.versions
+                                 ).match()
+                }
+            )
+        }
+    }
+
+    test("stub") {
```

Comment:
Naming convention: just same as the other tests and then add "- stub".
For example here it would be something like: 
```suggestion
    test("align_sequence - compressed - seatoxin - stub") {
```
if I understand it correctly.